types/aws-lambda/index.d.ts

```diff
@@ -252,6 +253,46 @@ export interface S3Event {
 }
 export type S3CreateEvent = S3Event; // old name
 
+/**
+ * S3 Batch Operations event
+ * https://docs.aws.amazon.com/AmazonS3/latest/dev/batch-ops-invoke-lambda.html
+ */
+export interface S3BatchEventRequest {
+    invocationSchemaVersion: string;
+    invocationId: string;
+    job: S3BatchEventJob;
+    tasks: S3BatchEventTask[];
+}
+
+export interface S3BatchEventJob {
+    id: string;
+}
+
+export interface S3BatchEventTask {
+    taskId: string;
+    s3Key: string;
+    s3VersionId?: string;
```

Comment:
This being optional makes me a little curious, as it isn't mentioned in the docs: is it completely not present for non-versioned buckets?

Reply:
yeah that's a really good point -- all my buckets are actually version enabled, let me whip something up

Reply:
@simonbuchan 

okay the results are in -- and you're correct, it's `null` -- i've committed a fix

actual payload:
```
            {
                "taskId": "AAAAAAAAAAHLSJmnbBR1fLJ5Y9/truncated",
                "s3BucketArn": "arn:aws:s3:::ommitted",
                "s3Key": "47600731-b366-4444-b0f4-24255c08c6e3.json",
                "s3VersionId": null
            }
```

Reply:
Thanks for checking!